conf/machine/raspberrypi3-64.conf

```diff
@@ -2,7 +2,7 @@
 #@NAME: RaspberryPi 3 Development Board
 #@DESCRIPTION: Machine configuration for the RaspberryPi 3 in 64 bits mode
 
-MACHINEOVERRIDES = "raspberrypi3:raspberrypi:${MACHINE}"
+MACHINEOVERRIDES = "raspberrypi3:${MACHINE}"
```

Comment:
It needs some more explanation. raspberrypi is a SOC family override and it natually would be used on all machines based on rpi. if its conflicting then we need to find a portable way to apply fixes for both pi0 and pi3-64

Reply:
The soc family is actually `rpi` and not `raspberrypi`.

Reply:
I realized that, but then raspberrypi is confusing, it is mean for rpi1 ?

Reply:
Yup. That's the confusing part. But calling it raspberrypi1 might be pretty strange too.

Reply:
It wont be since it falls in with the naming conventions that rpi boards have used but then they did not do it for the first board where 1 is sort of implicit. 

in anycase, I am fine with removing this rpi1 override from rpi3_64 machine overrides. Although it needs to go along with building and booting a rpi3_64 image

Reply:
Sure indeed. Anyway this was only for morty. We already have this change in master (and pyro afaik).

Reply:
OK I did not realize it was a cherry-pick, it sounded like a complete new patch